packages/@lightningjs/ui-components/src/components/TextBox/TextBox.test.js

```diff
@@ -168,14 +168,37 @@ describe('TextBox', () => {
       expect(textBox._Text.text.textColor).toBe(getValidColor('#000000'));
     });
 
-    // TODO: Fix reference to Base
-    // it('should get TextDefaults when switching themes', async () => {
-    //   context.setTheme(Base);
-    //   textBox.base = context.theme;
-    //   textBox.content = 'Hello Text Defaults';
-    //   await textBox.__updateSpyPromise;
-    //   expect(textBox._Text.text.maxLinesSuffix).toBe('..');
-    // });
+    it('should get TextDefaults when switching themes', async () => {
+      const testTheme = {
+        name: 'Test',
+        typography: {
+          display1: {
+            fontFace: 'TestFont1',
+            fontSize: 35,
+            fontWeight: 500,
+            letterSpacing: 1,
+            lineHeight: 100,
+            verticalAlign: 'middle'
+          },
+          body1: {
+            fontFace: 'TestFont2',
+            fontSize: 15,
+            fontWeight: 250,
+            letterSpacing: 0,
+            lineHeight: 50,
+            verticalAlign: 'middle'
+          }
+        }
+      };
+      context.setTheme(testTheme);
+      textBox.base = context.theme;
+      textBox.content = 'Hello Text Defaults';
+      await textBox.__updateSpyPromise;
+      expect(context.theme.name).toBe('Test');
+      expect(textBox.style.textStyle).toEqual({
+        ...context.theme.typography.body1
```

Comment:
I'm not sure if this is the right test case. Don't we want to confirm that textBox.style.textStyle matches testTheme.body1 to check that the theme was updated properly?